doc/indexing.rst

```diff
@@ -236,6 +236,7 @@ The :py:meth:`~xarray.Dataset.drop` method returns a new object with the listed
 index labels along a dimension dropped:
 
 .. ipython:: python
+    :okwarning:
 
     ds.drop(['IN', 'IL'], dim='space')
```

Comment:
Since this is deprecated, we should fix the docs to use `ds.drop(space=['IN', 'IL']` as an example of the preferred syntax going forward.